jonfroehlich/makeabilitylabwebsite
# Addresses Issue #257 (Deleting files locally

This PR addresses issue #257. Before, when model objects were deleted, the local files would not be deleted which uses up memory space and resources. 

Now, deleting Publications objects will also delete the pdf file and the generated thumbnail from the local file system. To prevent an error that results from a post_save signal that is triggered in signals.py, the automatic generation of the thumbnail was moved into an if statement that checks for the existence of the pdf_file. 

Deleting Talks objects will also delete the pdf file, the raw file, and or the thumbnail. The same adjustment was made to the signal function in signals.py. 

To test:
- Create a Talk and Publication object
- Check that their files are in the local file system under `media/talks` and `media/publications` respectively
- Delete both objects and check if the files still exist in the local file system
- Rinse and repeat five times with different files.
